### ISSUE TITLE
'yield_self' and 'then' missing from autocomplete

### ISSUE THREAD

**Issue by macifell** [castwide/solargraph#134]
I just noticed that `Kernel#yield_self` and it's alias `then` seem to be missing from the autocomplete.

**Comment by castwide** [comment]
Check which version of Ruby documentation you're using in Solargraph. The gem ships with 2.2.2, so that's probably what you're using if you haven't downloaded more. Some helpful commands:

```
$ solargraph list-cores      # List installed versions of docs
$ solargraph available-cores # List versions available for download
$ solargraph download-core   # Install docs that are closest to current Ruby version
```

IIUC, `then` is new in 2.6, so you won't see it in Solargraph yet (the highest doc version available is 2.5.3). `yield_self` has been there since 2.5, so it should show up if you `download-core`. I confirmed I get it in autocomplete with the 2.5.3 docs.

**Comment by macifell** [comment]
Awesome, I downloaded the most recent core and now I see `yield_self` and I understand why `then` doesn't show up yet. Thank you for explaining where it's coming from!
